### ISSUE TITLE
debug_print: Add support for build 985785

### ISSUE THREAD

**Issue by icuruok** [gibbed/Gibbed.Volition#2]
the patch 949013 and maybe a few others do not work with the "debug_print/xinput1_3.dll"

I believe this is due to some odd anti debugger placed within this patched version.

I need support added for patch version 985785

I need code support added for patch 985785 as I myself am unable to locate these locations within the games allocated virtual memory.

**Comment by gibbed** [comment]
Sure, I can look into that. Can't promise I'll get to it soon however.

**Comment by icuruok** [comment]
> Sure, I can look into that. Can't promise I'll get to it soon however.

Thank you so much ! 

My issue is I have patch version 949013 and patch version 985785
None of the debug_prints dll's work for patch 949013 due to its anti debug feature.

But it works fine for later patch versions but the only later patch version I have is
985785
Which sadly does not have support added into the code here for it.

I found a compiled version of your source here "idleninja" added support for
985785
https://www.saintsrowmods.com/forum/threads/debug_print-functionality-dll.554/

But sadly they did not provide the source or the memory addresses for this patch version.


hope you can help assist me with getting the correct allocated memory addresses for this patch.

Basically I require a custom re-compile of debug_print dll as I require it stream/write its message outputs to a file, This way I can easily allow external scripts/software to read the outputs from the debug console.
